tomviz/AlignWidget.cxx

```diff
@@ -709,10 +721,20 @@ void AlignWidget::applySliceOffset(int sliceNumber)
   {
     this->modes[i]->currentSliceUpdated(sliceNumber, offset);
   }
+  if (this->modes.length() > 0)
+  {
+    this->modes[this->currentMode]->update();
+  }
+  this->widget->update();
 }
 
 void AlignWidget::startAlign()
 {
+  if (!(this->frameRate > 0))
+  {
+    qDebug() << "Frame rate must be positive to start timer";
```

Comment:
I think we should favor opening up a warning rather than qDebug, but setting the minimum to 0, and counting a start with a frame rate of 0 as stop would seem reasonable and intuitive without a warning wouldn't it?